Insite_AngularUI/src/app/_components/attendance-list/attendance-list.component.html

```diff
@@ -0,0 +1,25 @@
+<div id="attendanceListContainer">
+    <h2>Attendance List</h2>
+
+    <mat-chip-list aria-label="Fish selection">
+        <mat-chip color="primary" value="3" selected>MCI vs LIV</mat-chip>
+        <mat-chip color="primary" value="4" selected>BOU vs ARS</mat-chip>
+        <mat-chip color="primary" value="5" selected>LIV vs AVA</mat-chip>
+        <mat-chip color="primary" value="12" selected>ARS vs MUN</mat-chip>
+        <mat-chip color="primary" value="3" selected>CRY vs EVE</mat-chip>
+        <mat-chip color="accent" value="S6" selected>2012 / 13</mat-chip>
+        <mat-chip color="accent" value="S7" selected>2016 / 17</mat-chip>
+        <mat-chip color="accent" value="S8" selected>2019 / 20</mat-chip>
```

Comment:
Ideally we would have liked you to set up an API controller for Games and Seasons, so that if I change the data the list of options changes.

The use of the `mat-chip-list` is a nice touch visually, but for this purpose the user will have to select both a season *and* game, which this setup does not allow for. Additionally, if the user can select from 38 games in multiple seasons, this type of view is quickly going to get overwhelming.